indexing-service/src/main/java/io/druid/indexing/overlord/RemoteTaskRunner.java

```diff
@@ -1316,22 +1304,76 @@ public ImmutableWorkerInfo apply(ZkWorker input)
   {
     return ImmutableList.copyOf(
         Collections2.transform(
-          workers,
-          new Function<ZkWorker, Worker>()
-          {
-            @Override
-            public Worker apply(ZkWorker input)
+            workers,
```

Comment:
Ditto here.. it was really tempting to replace everything with lambdas and stream operations but I was trying to be minimally invasive intentionally at this stage in the release cycle.